src/commands/deployment/get.ts

```diff
@@ -1,10 +1,29 @@
+import Table from "cli-table";
 import commander from "commander";
+import Deployment from "spektate/lib/Deployment";
 import { logger } from "../../logger";
-import {
-  getDeployments,
-  OUTPUT_FORMAT,
-  verifyAppConfiguration
-} from "./helper";
+import { verifyAppConfiguration } from "./init";
+import { clusterPipeline, config, hldPipeline, srcPipeline } from "./init";
```

Comment:
I think line 5 and 6 can be merged. If you are using all exported methods from ./init, _import *_ is another option.